ui/src/app/cohort-review/participant-status/participant-status.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import {ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import {ReactiveFormsModule} from '@angular/forms';
+import {ActivatedRoute} from '@angular/router';
+import {Observable} from 'rxjs/Observable';
+
+import {ParticipantStatusComponent} from './participant-status.component';
+
+import {CohortReviewService, CohortStatus} from 'generated';
+import {ReviewStateService} from '../review-state.service';
+
+import {updateAndTick} from '../../../testing/test-helpers';
+import {Participant} from '../participant.model';
+
+class StubRoute {
+    snapshot = {
+        params: {
+            ns: 'workspaceNamespace',
+            wsid: 'workspaceId',
+            cid: 1
+        },
+        data: {
+            workspace: {cdrVersionId: 1}
+        }
+    };
+}
+
+class ApiSpy {
+    updateParticipantCohortStatus = jasmine
+        .createSpy('updateParticipantCohortStatus')
+        .and
+        .returnValue(
+            Observable.of({participantId: 1, status: CohortStatus.INCLUDED, birthDate: 1}
+            ));
+}
+
+const PARTICIPANT: Participant =
+    new Participant({participantId: 1, status: CohortStatus.NOTREVIEWED, birthDate: 1});
+
+describe('ParticipantStatusComponent', () => {
+    let component:    ParticipantStatusComponent;
```

Comment:
NIT:  too much whitespace between argument and type

Reply:
Change indent to 2 spaces and reformatted file. Done.